storage/consistent_watchable_store.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storage
+
+import (
+	"log"
+	"strconv"
+)
+
+var (
+	indexBucketName = []byte("index")
+	indexKeyName    = []byte("index")
+)
+
+// IndexGetter is an interface that wraps the Get method.
```

Comment:
Can you please add a few sentence to explain what index is, since this interface is called IndexGetter.